conference/accounts.py

```diff
@@ -143,10 +144,17 @@ def social_connect_profile(backend, user, response, *args, **kwargs) -> None:
     """
 
     AssopyUser.objects.get_or_create(user=user)
+    attendee = get_or_create_attendee_profile_for_new_user(user=user)
+    return attendee
+
+
+def get_or_create_attendee_profile_for_new_user(user):
```

Comment:
This isn't idempotent, which a `get_or_create` should be - a new `uuid` is assigned when fetching a user. To make it idempotent, something like this would work:
```
    attendee, created = AttendeeProfile.objects.get_or_create(user=user)
    if not created:
        attendee.slug = slug_for_user(user)	    attendee.slug = slug_for_user(user)
        attendee.uuid = shortuuid.ShortUUID().random(length=7)
        attendee.save()
        attendee.setBio('bio')
    return attendee
```

Reply:
That's assuming the `setBio` call here is needed and it does a `save` - if it doesn't, it should be moved up.

Reply:
yes, it has to be there, and the dependency is actually not obvious, we could probably handle it in some other way, but this was/is the simplest one.